test/test_oneview_network_set.py

```diff
@@ -178,16 +191,16 @@ def test_update_when_only_bandwidth_has_modified_attributes(self):
         self.mock_ansible_module.exit_json.assert_called_once_with(
             changed=True,
             msg=NetworkSetModule.MSG_UPDATED,
-            ansible_facts=dict(network_set=DICT_PARAMS_WITH_CHANGES, connection_template=CONNECTION_TEMPLATE)
+            ansible_facts=dict(network_set=DICT_PARAMS_WITH_CHANGES)
         )
 
-    def test_update_when_data_has_modified_attributes_but_bandwidth_is_equal(self):
+    def test_update_when_data_has_modified_attributes_but_bandwith_is_equal(self):
```

Comment:
should be "test_update_when_data_has_modified_attributes_but_**bandwidth**_is_equal"

Reply:
done